Makefile

```diff
@@ -12,7 +12,7 @@ build:
 test: test-all
 
 test-all:
-	env MEMBER_COUNT=$(MEMBER_COUNT) go test $(TEST_FLAGS) $(PACKAGES) ./...
+	env MEMBER_COUNT=$(MEMBER_COUNT) HZ_VERSION=4.2.2 go test $(TEST_FLAGS) $(PACKAGES) ./...
```

Comment:
I think `HZ_VERSION` is left by mistake.

Reply:
Thanks for catching that, https://github.com/hazelcast/hazelcast-go-client/pull/755/commits/fc48bc504ba8d77e888bf8fa40c348e1616a0d9f